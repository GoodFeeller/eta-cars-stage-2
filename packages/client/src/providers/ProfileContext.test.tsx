import { render, screen } from "@testing-library/react"
import ProfileProvider from "./ProfileContext"

test('Profile context test', () => {
    render(<ProfileProvider ><div/></ProfileProvider>)
    const provider = screen.queryByTestId('provider')
    expect(provider).toMatchSnapshot()
})