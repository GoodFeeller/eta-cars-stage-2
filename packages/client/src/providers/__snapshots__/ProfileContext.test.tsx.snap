// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile context test 1`] = `null`;
